modules/managers/build-locally.sh

```diff
@@ -250,8 +250,20 @@ function update_release_yaml {
     fi
 }
 
+function check_if_script_invoked_directly() {
+    # Check if the script is being called directly or from another script
+    if [[ -z "${IN_CHAIN_MODE}" ]]; then
+        info "Script invoked directly, running detect-secrets.sh script"
+
+        # Run the detect-secrets.sh in root
+        cd "${WORKSPACE_DIR}/.."
+        TOOL_DIR=$(pwd)
+        $TOOL_DIR/tools/detect-secrets.sh
+    fi
+}
+
 
 build_code
 update_release_yaml
 
-check_secrets
```

Comment:
It was previously clcear that check-secrets was being called, now it's a side-effect of checking something ?
Suggest you just call the check_if_script_invoked_directly back to check_secrets_unless_supressed or something more meaningful.